docs/sources/release-notes/v2.2.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Grafana Mimir version 2.2 release notes"
+menuTitle: "V2.2 release notes"
+description: "Release notes for Grafana Mimir version 2.2"
+weight: 300
+---
+
+# Grafana Mimir version 2.2 release notes
+
+Grafana Labs is excited to announce version 2.2 of Grafana Mimir, the most scalable, most performant open source time series database in the world.
+
+Highlights include the top features, enhancements, and bugfixes in this release. If you are upgrading from Grafana Mimir 2.1, there is migration-related information as well.
+For the complete list of changes, see the [Changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Support for ingesting out-of-order samples**: Grafana Mimir includes new, experimental support for ingesting out-of-order samples.
+  This support is configurable, with users able to set how far out-of-order Mimir will accept samples on a per-tenant basis.
+  Note that this feature still needs a heavy testing, and is not production-ready yet.
+  For documentation see [Configuring out-of-order samples ingestion]({{< relref "../operators-guide/configuring/configuring-out-of-order-samples-ingestion.md" >}})
+
+- **Error messages**: The error messages that Mimir reports are more human readable, and the messages include error codes that are easily searchable.
+  For descriptions of errors, see the [Grafana Mimir runbooks / Errors catalog]({{< relref "../operators-guide/mimir-runbooks/_index.md#errors-catalog" >}}).
+
+- **Configurable prefix for object storage**: Mimir can now store block data, rules, and alerts in one bucket, each under its own user-defined prefix, rather than requiring one bucket for each.
+  You can configure the storage prefix by using `-<storage>.storage-prefix` option for corresponding storage: `ruler-storage`, `alertmanager-storage` or `blocks-storage`.
+
+- **Helm Chart update**: TBD
+
+- **Store-gateway can now optionally prepopulate the file system cache when memory-mapping index-header files.**
+  This can help store-gateway to avoid looking stuck while loading index-headers.
```

Comment:
```suggestion
  The store-gateway can now pre-populate the file system cache when memory-mapping index-header files. This can help the store-gateway avoid looking stuck while loading [index-headers]({{< relref "../operators-guide/architecture/binary-index-header" >}}). This feature is experimental and disabled by default; enable it using the flag `-blocks-storage.bucket-store.index-header.map-populate-enabled`.
```